endpoints/openrtb2/amp_auction.go

```diff
@@ -103,6 +103,7 @@ func NewAmpEndpoint(
 		storedRespFetcher,
 		hookExecutionPlanBuilder,
 		tmaxAdjustments,
+		openrtb_ext.NormalizeBidderName,
```

Comment:
Sorry. I hate to add yet another parameter here, but I can't test it properly otherwise.